control_dsf_plugins

```diff
@@ -0,0 +1,15 @@
+Package: ni-data-sharing-framework-plugins-support-veristand-{veristand_version}
```

Comment:
The NIPM convention is to put "support" after the product.

`Package: ni-data-sharing-framework-plugins-veristand-{veristand_version}-support`

Reply:
aaa5091e45817ab0702f5b58822a53a43aef2c0c